common/include/pcl/impl/cloud_iterator.hpp

```diff
@@ -197,8 +196,7 @@ namespace pcl
       }
 
       ~DefaultConstIterator ()
-      {
-      }
+      override = default;
```

Comment:
I would rather delete it entirely. Does clang-tidy provide options for that?

Reply:
Not explicitly, I'll change it to `= delete`

Reply:
I meant deleting the lines of code which explicitly define the destructor.

Reply:
I'm not sure what you're asking, but the diff shows that the `{}` was deleted and replaced by `= default`